app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -72,17 +67,17 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
           </tr>
         )}
         {experiment.jexlTargetingExpression &&
-          experiment.jexlTargetingExpression !== "" && (
-            <tr>
-              <th>Full targeting expression</th>
-              <td
-                data-testid="experiment-target-expression"
-                className="text-monospace"
-              >
-                <Code codeString={experiment.jexlTargetingExpression} />
-              </td>
-            </tr>
-          )}
+        experiment.jexlTargetingExpression !== "" ? (
+          <tr>
+            <th>Full targeting expression</th>
+            <td
+              data-testid="experiment-target-expression"
+              className="text-monospace"
+            >
+              <Code codeString={experiment.jexlTargetingExpression} />
+            </td>
+          </tr>
+        ) : null}
```

Comment:
This one's the extra console warning fixed:

![image](https://user-images.githubusercontent.com/13018240/118033989-13165e80-b32f-11eb-8f76-38a0bf1e22a3.png)

I'm actually not entirely sure why it apparently outputs an empty string/text node - I thought it was perhaps the indentation, but I turned off autoformat on save to play around with it without much success so our options seemed to be 1) remove the `experiment.jexlTargetingExpression` check and add a `!` to `codeString={experiment.jexlTargetingExpression}` to suppress TS complaints (👎 since this can be `null`) 2) explicitly render `null` in a ternary or 3) keep digging.